content/dev/blog/tidyeval.Rmd

```diff
@@ -0,0 +1,232 @@
++++
```

Comment:
Our hugo blog posts use TOML like this, but R Markdown may require YAML; if so, we'll need to munge this back to YAML for when we knit.

Also note that ta-da doesn't deal in Rmd so we'll have to knit to md ourselves.